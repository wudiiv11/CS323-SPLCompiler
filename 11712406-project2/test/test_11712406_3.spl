int fibo(int n) {
    if (n < 2) {
        return n;
    }
    return fibo(n-1) + fibo(n-2);
}

int main() {
    int c = 100;
    int x = fibo(5) + fibo(25);
    return c.num + fibo(c + x);
}