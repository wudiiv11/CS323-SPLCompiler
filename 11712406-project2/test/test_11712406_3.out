Error type 14 at Line 11: accessing with non-struct variable